### PR TITLE
[Remove Stream/Optional usage: portal-workflow-metrics-rest-impl]Lps 178425 3 30

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-178425

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:workflow

**Comment by slnn** [comment]
ci:test:workflow-metrics

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/TaskUtil.java [hunk 1]
```diff
@@ -223,51 +222,48 @@ public static Task toTask(
...
-		).collect(
-			Collectors.toMap(
-				entry -> _toLanguageTag(
-					StringUtil.removeSubstring(
-						entry.getKey(), fieldName + StringPool.UNDERLINE)),
-				entry -> GetterUtil.getString(entry.getValue()))
-		);
+				!StringUtil.endsWith(entry.getKey(), "_sortable")) {
+
+				hashMapWrapper.put(
```
Comment (dantewang) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/TaskUtil.java:258]: @slnn Missed this one for https://github.com/liferay-core-infra/liferay-portal/pull/4006#pullrequestreview-1360692502

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/SLAResourceImpl.java [hunk 2]
```diff
@@ -184,26 +183,30 @@ private long _getUserId() {
...
-					else {
-						executionType = nodeKeyStringParts[1];
-					}
+		return transform(
+			StringUtil.split(nodeKeysString),
+			nodeKey -> {
+				String[] nodeKeyStringParts = StringUtil.split(
+					nodeKey, StringPool.COLON);
+
+				if (!ArrayUtil.isEmpty(nodeKeyStringParts)) {
```
Comment (dantewang) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/SLAResourceImpl.java:192]: @slnn For such cases, please flip the if-condition to return null in if-body.

**Review by dantewang — CHANGES_REQUESTED** [review]
@slnn Please see my comment and double check all your commits.

**Pull request closed without merging**